webapp/channels/src/components/admin_console/filter/filter_checkbox.tsx

```diff
@@ -10,44 +10,42 @@ type Props = {
     updateOption: (checked: boolean, name: string) => void;
 };
 
-class FilterCheckbox extends React.PureComponent<Props> {
-    toggleOption = (e: React.MouseEvent) => {
+function FilterCheckbox(props: Props) {
+    const {name, checked, label} = props;
+
+    function toggleOption(e: React.MouseEvent) {
```

Comment:
Being purist, this should be a `useCallback` since it is passed to an underlying component, but since the underlaying component is a base one (`div`), let's leave it like this.

Reply:
I adjusted this to hopefully make it more consistent